protocol/lib/log/constants.go

```diff
@@ -46,9 +46,10 @@ const (
 	// Module tag values are prefixed with `x/`
 	Clob = "x/clob"
 
-	CheckTx   = "check_tx"
-	RecheckTx = "recheck_tx"
-	DeliverTx = "deliver_tx"
+	CheckTx        = "check_tx"
+	RecheckTx      = "recheck_tx"
+	DeliverTx      = "deliver_tx"
+	MsgBatchCancel = "msg_batch_cancel"
```

Comment:
is there a better group for this? doesn't seem related to CheckTx, DeliverTx, etc

Reply:
this is the batch for tag values, i just haven't been bothered to move anything over to this constants file other than the callbacks